spec/support/api/schemas/addresses/address.json

```diff
@@ -0,0 +1,24 @@
+{
+  "definitions":{
+    "address":{
+      "type": "object",
+      "properties":{
+        "id": { "type": "integer" },
+        "address_1": { "type": "string" },
+        "address_2": { "type": "string" },
+        "city": { "type": "string" },
+        "state": { "type": "string" },
+        "country": { "type": "string" },
+        "longitude": { "type": "float" },
+        "latitude": { "type": "float" },
+        "postal_code": { "type": "string" },
+        "active": { "type": "boolean" },
+        "formatted_address": { "type": "string" },
+        "company": {
+          "$ref": "../companies/company.json#/definitions/company"
+        }
+      },
+      "required": ["id", "address_1", "address_2", "city", "state", "country", "longitude", "latitude", "postal_code", "active", "formatted_address"]
```

Comment:
not really, for the specs I believe it should be however, it shouldn't affect the structural organization of the data